src/Sentry/Laravel/ServiceProvider.php

```diff
@@ -23,9 +24,10 @@ class ServiceProvider extends BaseServiceProvider
      * List of configuration options that are Laravel specific and should not be sent to the base PHP SDK.
      */
     private const LARAVEL_SPECIFIC_OPTIONS = [
-        // We do not want these settings to hit the PHP SDK because it's Laravel specific and the SDK will throw errors
+        // We do not want these settings to hit the PHP SDK because they are Laravel specific and the PHP SDK will throw errors
         'tracing',
         'breadcrumbs',
+        'tracing_integrations',
```

Comment:
Is there a reason why we want to separate this into tracing integrations? I would prefer to have it just as an integration to not intro something new.

Reply:
Good question actually... this was mostly to be able to keep it separate, all tracing code is registered using a second service provider in Laravel. If we add the tracing integrations as SDK integrations that would mean it needs to be added to the non-tracing service provider. 

But technically there is nothing stopping us from using the normal integrations. So we should probably do that!

Reply:
I would prefer to only have `intergrations` and enable this depending on `traces_sample_rate` option.

Reply:
Refactored it to do that 👍